src/registry_schemas/schemas/agm_extension.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/agm_extension",
+    "type": "object",
+    "definitions": {},
+    "title": "AGM Extension Information Schema",
+    "required": [
+        "agmExtension"
+    ],
+    "properties": {
+        "agmLocationChange": {
+            "type": "object",
+            "required": [
+                "year"

```

Comment:
should this be a date?
How does specifying a year (containing 12 months) have a condition of no more than 6 months. That doesn't align in my head.

Reply:
This is the year of the AGM they are requesting an extension for https://preview.uxpin.com/c32b7a6fabe9dea89e09a5ae63323f485a02396c#/pages/164523638/simulate/no-panels

It's not a date. It's just a year.

Reply:
Sounds incorrect, based on the business requirements below, sounds like dates need to be captured.
Four requirements for AGM extension:

Rules:
An association may apply for an extension in writing giving a brief explanation why the meeting must be delayed, when the meeting ought to have been held (within 4 months of the financial year end), and the date when the meeting will be held.
Have we already granted an extension for this AGM? The client may apply for a maximum six month extension, and not beyond March 31 of the year following the original AGM date. (They may apply twice, but the total extension cannot exceed six months.)
Is the coop in Good Standing? - see details below.
Check the prior years filed Annual Report to see if it had a reported meeting date on it.  If it was filed as "no meeting held" we will not provide an extension for the current year.
Has the request for the extension come in before the date the coop ought to have held the current year's AGM by? They must make the extension request before the day they ought to have held their AGM. A coop must hold a general meeting within 4 months after the end of its financial year.  If the request is coming later than 4 months after the financial year end it will be rejected.

Reply:
There will be more data captured but we haven't agreed on names yet.

THIS is an initial schema for the AGM extension filing. There is a part 2 ticket for completing the schema later.

In THIS schema, we do want to capture the year of the AGM that is being extended. That's this field.

Reply:
I have no further comment on this conversation. Thor, OK to resolve it?